mimic/model/maas_objects.py

```diff
@@ -189,32 +183,29 @@ def update(self, **kwargs):
         self.updated_at = int(1000 * kwargs['clock'].seconds())
 
 
-@attributes([Attribute('created_at', instance_of=int),
-             Attribute('details', default_factory=dict, instance_of=dict),
-             Attribute('id',
-                       default_factory=(lambda: u'nt' + random_hex_generator(4)),
-                       instance_of=text_type),
-             Attribute('label', default_value=u'', instance_of=text_type),
-             Attribute('metadata', default_factory=dict, instance_of=dict),
-             Attribute('type', default_value='email', instance_of=text_type),
-             Attribute('updated_at', instance_of=int)])
+@attr.s
 class Notification(object):
     """
     Models a MaaS Notification.
     """
+    created_at = attr.ib(validator=attr.validators.instance_of(int))
```

Comment:
These lines look pretty repetitive. Perhaps it would be better to `from attr.validators import instance_of` rather than re-typing that whole identifier each time? Or `an_int = attr.validators.instance_of(int)` up at the top so each line only says `validator=an_int` ?